spam/spam_data_augmentation_tutorial.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
```

Comment:
Overall comment- I think these tutorials should be numbered so it's clear how to proceed through them.  This can be in the name (e.g. `02_spam_data_augmentation...`) and/or in the README, but should be very clear somewhere